src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -5,10 +5,57 @@
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int MIN_LENGTH = 6;
```

Comment:
lets have two constants for each length

Reply:
Can you explain? I need only one constant, to limit minimal length of password & login. For what i need second one? I don't understand

Reply:
Ohh, i understand now. I need to use two constants, one for length limit in login, second for password. Thanks!